### DISCUSSION TITLE
Swift Library

### DISCUSSION THREAD

**Discussion by atacan** [catppuccin/catppuccin#2030] [General]
Hello, I created a Swift package https://github.com/atacan/swift-catpuccin/

Let me know what you think. 

I see that other libraries (python, go etc.) are under catpuccin project. What is your approach on this?

**Comment by vanillyn** [comment]
You should follow the [catppuccin repo template](https://github.com/catppuccin/template)
You should also follow the [other standards](https://github.com/catppuccin/catppuccin/blob/main/docs/port-creation.md#creation), such as all the previews being in `assets/` and using `.webp`/`.webm`
